content/en/docs/getting-started/running/index.md

```diff
@@ -4,54 +4,63 @@ description: Operating and Managing Falco
 weight: 4
 ---
 
+## Falco packages
 
-## Run Falco as a service
+If you installed the Falco packages using the `dialog` all your services should be already up and running, while if you chose the `Manual configuration` or if you used the `FALCO_FRONTEND=noninteractive` env variable you have to configure services by your hand. Here we show a simple example with the `eBPF probe`.
 
-If you installed Falco by using [the DEB or the RPM](/docs/getting-started/installation) package, then falco systemd service was already started and enabled for you.  
-In case you wish to stop or disable it, issue:
+Let's imagine we want to start the `falco-bpf.service`.
 
+1. Type `systemctl list-units | grep falco` to check that no unit is running.
 
-```console
-systemctl disable falco
-```
+2. Now you have to decide if you want the Falcoctl service running together with the Falco one or not. If yes you don't have to do anything, if no you have to mask the Falcoctl service with `systemctl mask falcoctl-artifact-follow.service`. As said [in this section](../installation.md#rule-update) the Falcoctl service is strictly related to the Falco one so if you don't mask it, it will be started together with the Falco service.
```

Comment:
```suggestion
2. Now you have to decide whether you want the Falcoctl service running together with the Falco one. If yes you don't have to do anything, else you will need to mask the Falcoctl service with `systemctl mask falcoctl-artifact-follow.service`. As pointed out [in this section](../installation.md#rule-update) the Falcoctl service is strictly related to the Falco one so if you don't mask it, it will be started together with the Falco service.
```